cnd/src/network/comit/swaps.rs

```diff
@@ -306,265 +316,58 @@ impl Default for Swaps<()> {
 mod tests {
     use super::*;
     use crate::{
-        asset, asset::ethereum::FromWei, identity, network::DialInformation, swap_protocols::Role,
+        asset::{self, ethereum::FromWei},
+        identity,
+        network::LocalData,
     };
+    use ::comit::network::{protocols::announce::SwapDigest, swap_digest::Herc20Halight};
     use digest::Digest;
 
-    impl<T> Swaps<T> {
-        /// Gets a swap that was announced
-        pub fn get_announced_swap(
-            &self,
-            local_swap_id: &LocalSwapId,
-        ) -> Option<(SharedSwapId, Herc20HalightBitcoinCreateSwapParams)> {
-            let create_params = match self.swaps.get(local_swap_id) {
-                Some(create_params) => create_params,
-                None => return None,
-            };
-
-            let shared_swap_id = match self.swap_ids.get(local_swap_id) {
-                Some(shared_swap_id) => shared_swap_id,
-                None => return None,
-            };
-
-            Some((*shared_swap_id, create_params.clone()))
-        }
-    }
-
-    fn create_params() -> Herc20HalightBitcoinCreateSwapParams {
-        Herc20HalightBitcoinCreateSwapParams {
-            role: Role::Alice,
-            peer: DialInformation {
-                peer_id: PeerId::random(),
-                address_hint: None,
-            },
-            ethereum_identity: identity::Ethereum::random(),
+    // FIXME: Usage of this function relies on the fact that token_contract is
```

Comment:
Feel free to remove the comment without actioning.

Reply:
Cool, I did that already :)